test/integration.test.ts

```diff
@@ -150,4 +154,44 @@ describe('Starting cardano-wallet (and its node)', () => {
       }),
     longTestTimeoutMs
   );
+
+  describe('Child process logging support', () => {
+    const tmpDir = './test/.tmp';
+    const logFile = path.join(tmpDir, 'logs');
+
+    beforeEach(async () => {
+      await ensureDir(tmpDir);
+      await remove(logFile);
+      await writeFile(logFile,'');
+    });
+
+    afterEach(async () => {
+      await remove(tmpDir)
+    });
+
+    it('Accepts a WriteStream, and pipes the child process stdout and stderr streams', async () => {
+      const childProcessLogWriteStream = createWriteStream(logFile);
+      const statsBefore = await stat(logFile);
+      expect(statsBefore.mtimeMs === statsBefore.birthtimeMs);
+      const launcher = new Launcher({
+        stateDir:(
+            await tmp.dir({
+              unsafeCleanup: true,
+              prefix: 'launcher-integration-test-2',
+            })
+        ).path,
+        networkName: 'self',
+        nodeConfig: {
+          kind: 'jormungandr',
+          configurationDir: path.join('test', 'data', 'jormungandr'),
+          network: jormungandr.networks.self,
+        },
+        childProcessLogWriteStream
+      });
+      await launcher.start();
+      const statsAfter = await stat(logFile);
+      expect(statsAfter.mtimeMs > statsAfter.birthtimeMs);
```

Comment:
This check doesn't seem very portable to me. I believe some Linux file systems only have mtime resolution of 1 second. On Windows it depends on whether it's [NTFS or FAT](https://docs.microsoft.com/en-us/windows/win32/sysinfo/file-times)...
Checking the file size should work. I believe the logging framework flushes the log file pretty regularly.